doc/user/zebra.rst

```diff
@@ -445,7 +465,8 @@ commands in relationship to VRF. Here is an extract of some of those commands:
    This command is available on configuration mode. By default, above command
    permits accessing the vrf configuration mode. This mode is available for
    both VRFs. It is to be noted that *Zebra* does not create *Linux VRF*.
```

Comment:
I know this line isn't changed but `Linux VRF` doesn't need to be in *emphasis*.

Reply:
ok